R/CriteriaMethods.R

```diff
@@ -0,0 +1,436 @@
+#' Define Criteria and Methodology
+#'
+#' Users will need to provide the completed reference tables from
+#' TADA_CreateSpatialRef. This will generate a template for users to fill out
+#' and define either the full Criteria or magnitude only values associated with
+#' an ATTAINS Parameter name and use name. For each Criteria/Magnitude value,
+#' users will need to ensure they properly define any additional methods that will
+#' correctly reflect their assessment standards for that parameter and use.
+#' For example, if there are separate standards for acute versus chronic,
+#' rivers versus estuary, different seasons, etc. then a user will need to create
+#' additional rows to reflect this.
+#'
+#' Efforts have been made to pull in the EPA304a recommended standards
+#' automatically from the Criteria Search Tool (CST). Users should validate this
+#' final output if a user has decided to include the EPA304a standards.
+#' User will need to determine if any additional adjustments are needed.
+#' For example, does the crosswalk between the EPA304A.PollutantName
+#' and TADA.ComparableDataIdentifier seem valid for your organization's method?
+#' Is your organization only interested in providing the EPA304a recommended
+#' standards for certain seasons (Fall, Summer, Spring, Winter) etc.
+#'
+#' @param excel A Boolean value that returns an excel spreadsheet if
+#' excel = TRUE. This spreadsheet is created in the user's downloads folder path.
+#' If you have any trouble locating the file, please type the following into
+#' your R console to locate it: file.path(Sys.getenv("USERPROFILE"), "Downloads").
+#' The file will be named "myfileRef.xlsx". The excel spreadsheet will highlight
+#' the cells in which users should input information.
+#'
+#' @param overwrite A Boolean value that ensures the function will not overwrite
+#' the user supplied crosswalk entered into this function via the paramRef
+#' function input. This helps prevent users from overwriting their progress.
+#'
+#' @param .data A TADA dataframe. Users should run the appropriate data cleaning,
+#' processing, harmonization and filtering functions prior to this step.
+#'
+#' @param spatialRef An optional data frame which contains the completed spatial
+#' crosswalk to assign any unique spatial criteria to a parameter, use, waterbody
+#' or monitoring site/assessment unit.
+#' 
+#' @param ref "TADA"
```

Comment:
Can you add a definition for the ref = "TADA" input and an example that uses it?